src/main/java/seedu/medibook/model/medicalnote/MedicalNoteList.java

```diff
@@ -4,18 +4,21 @@
 import java.util.LinkedList;
 import java.util.List;
 
+import javafx.collections.FXCollections;
+import javafx.collections.ObservableList;
+
 /**
  * Represents a list of medical notes for a Patient in MediBook.
  */
 public class MedicalNoteList implements ReadOnlyMedicalNoteList {
     private static final MedicalNoteComparator comparator = new MedicalNoteComparator();
-    private final List<MedicalNote> innerList;
+    private final ObservableList<MedicalNote> innerList;
```

Comment:
Maybe add a JavaDoc comment (single line "/** comment here */) to describe innerList?

Reply:
Resolved